integration_tests/src/main/python/parquet_test.py

```diff
@@ -728,3 +729,44 @@ def do_parquet_scan(spark):
         exist_classes= "BatchScanExec",
         non_exist_classes= "GpuBatchScanExec",
         conf = conf_for_parquet_aggregate_pushdown)
+
+@pytest.mark.skipif(is_before_spark_330(), reason='Hidden file metadata columns are a new feature of Spark 330')
```

Comment:
It would be good to have a smoke test for metadata columns in the ORC and CSV tests as well, as that should be covered by the updated changes.